### PR TITLE
fix issue #1 Histogram for rank of logits not working when S >= 10

### PR DESCRIPTION
**Pull request by nocluebutalotofit**
Using the formula in the paper w_bar = r / (N+1) instead of w_bar = r / N avoids the occurance of 1.0 in w_bar which leads to inf in logits. Intuitively, just because all of the samples have outperformed one cannot be 100% sure.

inf in logits prevented sns.distplot() to plot the KDE. The higher S the more likely this situation is encountered.

### CODE REVIEW

**Comment by esvhd** [comment]
Thanks, merged.
